src/Foundation/Accounts/code/Services/ContactFacetService.cs

```diff
@@ -200,18 +200,7 @@ private bool SetEmail(ContactFacetData data, XConnect.Contact contact, XConnectC
                 return false;
             }
             var emails = contact.Emails();
-            if (emails == null)
```

Comment:
Hey @scottmulligan 
Looks like you're setting e-mails to contact.Emails() but immediately overwriting it with new EmailAddressList. Are you now assuming that the contact will never have an existing e-mail?

If the business logic is fine maybe ermove the 
`var emails = contact.Emails();`
and add the declaration to the next line:
`var emails = new EmailAddressList(new EmailAddress(email, true), "Work Email");`